runtime/src/lib.rs

```diff
@@ -360,20 +389,87 @@ parameter_types! {
 
 impl pallet_balances::Config for Runtime {
 	type MaxLocks = ConstU32<50>;
+	type MaxReserves = ();
+	type ReserveIdentifier = [u8; 8];
 	/// The type for recording an account's balance.
 	type Balance = Balance;
 	/// The ubiquitous event type.
 	type RuntimeEvent = RuntimeEvent;
 	type DustRemoval = ();
-	type ExistentialDeposit = ExistentialDeposit;
+	type ExistentialDeposit = ConstU128<EXISTENTIAL_DEPOSIT>;
 	type AccountStore = System;
-	type WeightInfo = pallet_balances::weights::SubstrateWeight<Runtime>;
-	type MaxReserves = ConstU32<50>;
-	type ReserveIdentifier = [u8; 8];
-	type RuntimeHoldReason = RuntimeHoldReason;
-	type RuntimeFreezeReason = RuntimeFreezeReason;
+	type WeightInfo = weights::pallet_balances::WeightInfo<Runtime>;
 	type FreezeIdentifier = [u8; 8];
-	type MaxFreezes = ConstU32<0>;
+	type MaxFreezes = ();
+	type RuntimeHoldReason = ();
+	type RuntimeFreezeReason = ();
+}
+
+parameter_types! {
+    pub const InclusionFeesDistributionKey: DistributionKey = DistributionKey {
+        legal_officers_percent: Percent::from_percent(35),
+        community_treasury_percent: Percent::from_percent(30),
+        logion_treasury_percent: Percent::from_percent(35),
+        loc_owner_percent: Percent::from_percent(0),
+    };
+
+	// Inflation: I=0,05 (5%)
+	// Total supply: N=10^9
+	// Block rate: B=6 (Number of seconds between 2 blocks)
```

Comment:
Block rate is 12 on the parachain so `InflationAmount` should be adjusted to take this into account.